src/vs/workbench/browser/layout.ts

```diff
@@ -14,7 +14,6 @@ import { IViewletService } from 'vs/workbench/services/viewlet/browser/viewlet';
 import { IStorageService, StorageScope } from 'vs/platform/storage/common/storage';
```

Comment:
Changes in this file make sense. You are using titlebarheight instead of headingHeight since we are mergin now these elements.

Reply:
T